build/alpine-iptables/README.md

```diff
@@ -0,0 +1,32 @@
+### alpine-iptables
+
+Serves as the base image for `gcr.io/google_containers/kube-proxy-${ARCH}` and multiarch (not `amd64`) `gcr.io/google_containers/flannel-${ARCH}` images.
```

Comment:
The flannel images don't exist anymore, the flannel team took care of it themselves after a while